deltametrics/utils.py

```diff
@@ -218,3 +201,280 @@ def func_linear(x, a, b): return a * x + b
     perror = np.sqrt(np.diag(pcov))
 
     return yfit, pcov, perror
+
+
+def guess_land_width_from_land(land_col_0):
```

Comment:
I know this is a 'temporary' method until we add the land width to the netCDF file, but could this be called during cube initialization and held in the cube as `self._trim_idx` or something similar? Might make the swap down the line easier to implement as the function would be called once during initialization instead of spread across multiple files/methods?

Reply:
Probably a good idea, and related to my bug on the `RadialSection` too. I'm going to defer to after updating the netcdf file though, to see what the best way to pull this info into the `Cube` is. I think probably creating some attributes of the cube based on whether those attributes exist in the netcdf file will make the most sense. For example, if `L0` is in the netcdf, then make `self.L0` in the `Cube` and then when creating a `Section`, you can check whether `Cube.L0` exists, and use it if it does. 

I'm going to open an issue for this.